### ISSUE TITLE
Modal / no component for CID

### ISSUE THREAD

**Issue by maennchen** [phoenixframework/phoenix_live_view#1199]
### Environment

* Elixir version (elixir -v): `1.11.0`
* Phoenix version (mix deps): `1.5.6`
* Phoenix LiveView version (mix deps): `master` / `597c5ddf8af2ca39216a3fe5a44c066774de3abd`
* NodeJS version (node -v): `12.18.3`
* NPM version (npm -v): `6.14.6`
* Operating system: `Linux tardis 5.0.0-32-generic #34-Ubuntu SMP Wed Oct 2 02:06:48 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`
* Browsers you attempted to reproduce this bug on (the more the merrier): Chrome 86
* Does the problem persist after removing "assets/node_modules" and trying again? Yes/no: Yes

### Actual behavior

* Create LiveView with `mix phx.gen.live`
* Open a modal (for example for create)
* Close the modal
* Open another modal
* See error in console

```
phoenix_live_view.js?28bb:1 no component for CID 1 {}
K @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
eval @ phoenix.js?2bc0:1
value @ phoenix.js?2bc0:1
eval @ phoenix.js?2bc0:1
value @ phoenix.js?2bc0:1
eval @ phoenix.js?2bc0:1
value @ phoenix.js?2bc0:1
eval @ phoenix.js?2bc0:1
decode @ phoenix.js?2bc0:1
value @ phoenix.js?2bc0:1
conn.conn.onmessage @ phoenix.js?2bc0:1


phoenix_live_view.js?28bb:1 Uncaught TypeError: Cannot read property 'd' of undefined
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
    at e.value (phoenix_live_view.js?28bb:1)
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
value @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
eval @ phoenix_live_view.js?28bb:1
eval @ phoenix.js?2bc0:1
value @ phoenix.js?2bc0:1
eval @ phoenix.js?2bc0:1
value @ phoenix.js?2bc0:1
eval @ phoenix.js?2bc0:1
value @ phoenix.js?2bc0:1
eval @ phoenix.js?2bc0:1
decode @ phoenix.js?2bc0:1
value @ phoenix.js?2bc0:1
conn.conn.onmessage @ phoenix.js?2bc0:1
```

### Expected behavior

No Error

**Comment by maennchen** [comment]
This is fixed on the lastest master commit (`7e515e7e664bf0eb074388d884297662b3731264`), sorry for the noise.

**Comment by josevalim** [comment]
Thank **you** for checking master. We appreciate it, it saves us a lot of time!
